akka-persistence-typed/src/main/scala/akka/persistence/typed/dstate/DurableStateSignal.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.dstate
```

Comment:
I had that thought as well, but then my conclusion was that the name of the class would be enough to make it clear that it's not about distributed state, but durable state. 

So, we have `ddata.DistributedData` and `dstate.DurableStateBehavior`. 

I didn't pick `state` because the final package would be `akka.persistence.typed.state`. Unless we make `akka.persistence.state.typed`. 

Anyway, I think we need to let it sink a little bit more.